apis/swagger.yml

```diff
@@ -4159,33 +4159,6 @@ definitions:
       Link:
         type: "boolean"
 
-  ContainerListOptions:
```

Comment:
Good catch! Actually we have used this struct. Also described in https://github.com/alibaba/pouch/issues/2400.

Reply:
Maybe the not used warning was caused by not use in swagger.yaml, such as `$ref: '#/definitions/'`?
In addition, it's a question put this kind of struct into swagger or just define in the go file.